### PR TITLE
Small refactor on ipfs proposal description hash

### PR DESCRIPTION
**Pull request by AugustoL**
- Supports future v1 ipfs hashes.
- Optimize a bit the description hash handling.

### CODE REVIEW

File: src/services/IPFSService.ts [hunk 1]
```diff
@@ -49,15 +49,18 @@ export default class IPFSService {
...
-      'https://davi.mypinata.cloud/ipfs/',
-      'https://ipfs.io/ipfs/',
-      'https://gateway.ipfs.io/ipfs/',
-      'https://cloudflare-ipfs.com/ipfs/',
-      'https://dweb.link/ipfs/',
-      'https://infura-ipfs.io/ipfs/',
-    ];
+    const gatewayURLBaseList =
+      hash.substring(0, 2) == 'Qm'
+        ? [
```
Comment (rossneilson) [inline src/services/IPFSService.ts:54]: These should support v1 I think

**Comment by rossneilson** [comment]
This wont support the v1 already in mainnet? because its not working so testing is tough

File: src/services/IPFSService.ts (hunk 1 above)
Comment (AugustoL) [reply src/services/IPFSService.ts:54]: It does, is the other option, if it doesn't start with Qm it is v1.

**Comment by AugustoL** [comment]
@rossneilson I cleared cache and only one (from two using v1 setting ens) proposal title was caught in the proposals page,

**Review by rossneilson — APPROVED** [review]

**Pull request merged**